test/e2e/storage/empty_dir_wrapper.go

```diff
@@ -181,7 +191,13 @@ var _ = utils.SIGDescribe("EmptyDir wrapper volumes", func() {
 		}
 	})
 
-	It("should not cause race condition when used for git_repo [Serial] [Slow]", func() {
+	// SLow by design [~150 Seconds]
+	/*
+		Release : v1.13
+		Testname: EmptyDir Wrapper Volume, GitRepo volumes, no race
+		Description: Create a Git Server service exposing URL and repo. 50 GitRepo Volumes are created with this Git Repository. 5 replicas of pod is created mounting GitRepo Volumes. Pod MUST NOT fail waiting for Volumes.
+	*/
+	framework.ConformanceIt("should not cause race condition when used for git_repo [Serial] [Slow]", func() {
```

Comment:
I don't think we should make a conformance test for gitRepo, which is deprecated

Reply:
Oh actually i think the test was changed to not use the git repo volume.  We should update the test name and description to reflect that

Reply:
Thanks for summarizing! I think what you said makes sense.  Let's drop the git_repo race test from conformance, but promote the other 2.  Maybe you can also add a comment to the git_repo test explaining why we can't promote

Reply:
Done.